crates/cheatcodes/src/toml.rs

```diff
@@ -0,0 +1,214 @@
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Uint(256))
+    }
+}
+
+impl Cheatcode for parseTomlIntCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Int(256))
+    }
+}
+
+impl Cheatcode for parseTomlIntArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Int(256))
+    }
+}
+
+impl Cheatcode for parseTomlBoolCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Bool)
+    }
+}
+
+impl Cheatcode for parseTomlBoolArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Bool)
+    }
+}
+
+impl Cheatcode for parseTomlAddressCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Address)
+    }
+}
+
+impl Cheatcode for parseTomlAddressArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Address)
+    }
+}
+
+impl Cheatcode for parseTomlStringCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::String)
+    }
+}
+
+impl Cheatcode for parseTomlStringArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::String)
+    }
+}
+
+impl Cheatcode for parseTomlBytesCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Bytes)
+    }
+}
+
+impl Cheatcode for parseTomlBytesArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::Bytes)
+    }
+}
+
+impl Cheatcode for parseTomlBytes32Call {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::FixedBytes(32))
+    }
+}
+
+impl Cheatcode for parseTomlBytes32ArrayCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_coerce(&convert(toml)?, key, &DynSolType::FixedBytes(32))
+    }
+}
+
+impl Cheatcode for parseTomlKeysCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { toml, key } = self;
+        parse_json_keys(&convert(toml)?, key)
```

Comment:
We can dedup the `parse_json(convert()?)` to their own parse_toml functions, wdyt?